stdlib/arg.mli

```diff
@@ -186,12 +186,12 @@ val read_arg: string -> string array
 
 val read_arg0: string -> string array
 (** Identical to {!Arg.read_arg} but assumes NUL terminated command line
```

Comment:
I know it's not part of this diff, but this should really be `NUL-terminated` (however, the manual and the docs are very inconsistent about null/NULL/NUL!)